russian/light-fire/open-the-door/look_around/look_around.md

```diff
@@ -0,0 +1 @@
+Снаружи дом выглядит вполне обычно. Вы проходите сотню метров вдоль улицы. Впереди слышен бодрый марш. Вы замечаете, что у большинства мужчин серьга в одном ухе и единственная перчатка на разноименной руке.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.